chanjo_report/server/blueprints/report/views.py

```diff
@@ -101,7 +103,12 @@ def pdf():
 
     # check if the request is to download the file right away
     if 'dl' in request.args:
```

Comment:
Shouldn't this be data_dict now rather to cater for both GET and POST?

Reply:
Good point, I'll change it!